app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,24 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
- *= require_self
  */
+
```

Comment:
Don't use 'application.css' for styles.

Reply:
@svidlonix  I don't know what should I use